scripts/cli/lec.sh

```diff
@@ -485,14 +506,35 @@ _cmd_gw() {
 		./gradlew "${@}"
 	)
 }
-_cmd_fn() {
-	"${1}" "${@:2}"
-}
 _cmd_list() {
-	_listWorktrees
-}
-_cmd_listRunning() {
-	_listRunningProjects
+	local closest_resource
+	local resource="${1}"
+
+	if [[ ${resource} == "" ]]; then
```

Comment:
2 things here:

1. Variable expansion should generally be quoted to avoid word splitting: 
```sh
if [[ "${resource}" == "" ]]; then
```

2. Using `-z` is more concise than comparing to an empty string:
```sh
if [[ -z "${resource}" ]]; then
```